templates/fargate-service.cfn.yml

```diff
@@ -79,23 +79,23 @@ Parameters:
     MaxLength: 255
     ConstraintDescription: Value must be between 1 and 255 characters
 
-  GitHubSourceRepo:
+  GitSourceRepo:
     Type: String
-    Description: GitHub source repository - must contain a Dockerfile in the base
+    Description: CodeCommit or GitHub source repository - must contain a Dockerfile in the base
```

Comment:
Examples here might be useful, first time I used this was unclear to me if it should be the 'fully qualified' repo name, ie 'rnzsgh/startup-kit-templates' vs 'startup-kit-templates'